src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +38,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
I'm not really sure about this use-case, honestly  :wink: but I see your point…
Then what about interpolating the `href` so that the destination URL is the filled search?  :)  Maybe the `props.history` API exposes a way to get the previous URL?